src/views/albums/Album.vue

```diff
@@ -61,27 +61,33 @@
 </template>
 
 <script>
-import { mapState } from "vuex";
+import { mapState, mapActions } from "vuex";
 import AlbumActions from "../../components/AlbumActions";
 import TracksTable from "../../components/TracksTable";
 import AlbumArtists from "../../components/AlbumArtists";
+import { TracksScope } from "@/api/scopes";
 
 export default {
   name: "Album",
   components: { AlbumArtists, TracksTable, AlbumActions },
   metaInfo() {
     return { title: this.album?.title };
   },
+  props: {
+    id: {
+      type: [String, Number],
```

Comment:
The `[String, Number]` feels weird to me. Will this not always be a `String` since it is loaded from the URL? Or does `vue-router` only read the prop from the URL if it was not passed in code?

Reply:
The latter. If you go to `app/albums/1` directly the prop will be a string (and Vue's type checker will complain if that is not an option), if you navigate through the app the prop will be a number.

Reply:
OK.